docs/source/usersguide/output/summary.rst

```diff
@@ -0,0 +1,228 @@
+.. _usersguide_summary:
+
+===================
+Summary File Format
+===================
+
+The current revision of the summary file format is 1.
+
+**/filetype** (*char[]*)
+
+    String indicating the type of file.
+
+**/revision** (*int*)
+
+    Revision of the summary file format. Any time a change is made in the
+    format, this integer is incremented.
+
+**/version_major** (*int*)
+
+    Major version number for OpenMC
+
+**/version_minor** (*int*)
+
+    Minor version number for OpenMC
+
+**/version_release** (*int*)
+
+    Release version number for OpenMC
+
+**/date_and_time** (*char[]*)
+
+    Date and time the state point was written.
+
+**/n_procs** (*int*)
+
+    Number of MPI processes used.
+
+**/n_particles** (*int8_t*)
+
+    Number of particles used per generation.
+
+**/n_batches** (*int*)
+
+    Number of batches to simulate.
+
+if (run_mode == MODE_EIGENVALUE)
+
+    **/n_inactive** (*int*)
+
+        Number of inactive batches.
+
+    **/n_active** (*int*)
+
+        Number of active batches.
+
+    **/gen_per_batch** (*int*)
+
+        Number of generations per batch.
+
+end if
+
+**/geometry/n_cells** (*int*)
+
+**/geometry/n_surfaces** (*int*)
+
+**/geometry/n_universes** (*int*)
+
+**/geometry/n_lattices** (*int*)
+
+do i = 1, n_cells
+
+    **/geometry/cells/cell <uid>/index** (*int*)
+
+    **/geometry/cells/cell <uid>/name** (*char[]*)
+
+    **/geometry/cells/cell <uid>/universe** (*int*)
+
+    **/geometry/cells/cell <uid>/fill_type** (*char[]*)
+
+    **/geometry/cells/cell <uid>/material** (*int*)
+
+    **/geometry/cells/cell <uid>/maps** (*int*)
+
+    **/geometry/cells/cell <uid>/offset** (*int[]*)
+
+    **/geometry/cells/cell <uid>/translated** (*int*)
```

Comment:
Problem is that booleans don't exist in HDF5. I'm guessing this is because C89 had no boolean type. We could write an unsigned short integer, but it's still going to be a zero or one.